src/Sitemap/Generators/LlmsTxtGenerator.php

```
@@ -0,0 +1,448 @@
+	}
+
+	/**
+	 * Load indexable sitemap entries, applying type include/exclude rules.
+	 *
+	 * @since 1.4.0
+	 *
+	 * @return Collection<int, SitemapEntry>
+	 */
+	protected function getEntries(): Collection
+	{
+		$query = SitemapEntry::indexable()
```

PHPStan (2.2.8)

[error] Call to an undefined static method ArtisanPackUI\SEO\Models\SitemapEntry::indexable().


(staticMethod.notFound)